feature-libs/product/future-stock/components/future-stock-trigger/future-stock-trigger.component.ts

```diff
@@ -1,13 +1,44 @@
 import {
   ChangeDetectionStrategy,
   Component,
+  OnInit,
 } from '@angular/core';
-
+import { FutureStock } from '../../root/models/future-stock.model';
 
 @Component({
   selector: 'cx-future-stock',
   templateUrl: './future-stock-trigger.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class FutureStockTriggerComponent {
+export class FutureStockTriggerComponent implements OnInit {
```

Comment:
I'm thinking if "trigger" is a good word. You meant by this more like a wrapper for your accordion right?
Can this one be called just like `FutureStockComponent`?

Reply:
good point, done